README

```diff
@@ -1297,6 +1304,104 @@ apex71: stopping org.apache.spark.deploy.worker.Worker
 apex76: stopping org.apache.spark.deploy.worker.Worker
 stopping org.apache.spark.deploy.master.Master
 
+Instructions For Using Phoenix
+--------------------------
+
+0) If necessary, download your favorite version of Phoenix off of Apache
+   and install it into a location where it's accessible on all cluster
+   nodes.  Usually this is on a NFS home directory.
+
+   Make sure that the version of Phoenix you install is compatible with
+   the version of Hbase you are using.
+
+   In some cases, a re-compile of Phoenix may be necessary.  For example,
+   by default Phoenix 4.5.2 works against the .98, 1.0 and 1.1 versions
+   of Hbase. You may need to recompile to use other version of Hbase.
+
+   See below about misc/magpie-apache-download-and-setup.sh, which may
+   make the downloading easier.
+
+1) Select an appropriate submission script for running your job.  You
+   can find them in the directory submission-scripts/, with Slurm
+   Sbatch scripts using srun in script-sbatch-srun, Moab Msub+Slurm
+   scripts using srun in script-msub-slurm-srun, Moab Msub+Torque
+   scripts using pdsh in script-msub-torque-pdsh, and LSF scripts
+   using mpirun in script-lsf-mpirun.
+
+   You'll likely want to start with the base script
+   (e.g. magpie.sbatch-srun) for your scheduler/resource manager.  
+
```

Comment:
Shouldn't this be magpie.sbatch-srun-hbaes-with-hdfs-with-phoenix?  This is what I had for hadoop

```
   You'll likely want to start with the base hadoop script
   (e.g. magpie.sbatch-srun-hadoop) for your scheduler/resource
   manager.  If you wish to configure more, you can choose to start
   with the base script (e.g. magpie.sbatch-srun) which contains all
   configuration options.
```